specification/testbase/resource-manager/Microsoft.TestBase/preview/2023-11-01-preview/actionrequest.json

```diff
@@ -0,0 +1,375 @@
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TestBaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RequestTypeFilter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TopFilter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Succeeded to list all action requests under the specified test base account.",
+            "schema": {
+              "$ref": "#/definitions/ActionRequests"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ActionRequestList": {
+            "$ref": "./examples/ActionRequestList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActionRequests": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActionRequest"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next set of results. Not empty if value contains incomplete list of results."
+        }
+      }
+    },
+    "ActionRequest": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ActionRequestProperties",
+          "x-ms-client-flatten": true,
+          "description": "Test Base Draft Package properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "ActionRequestProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "$ref": "./testbase.json#/definitions/ProvisioningState"
+        },
+        "requestType": {
+          "$ref": "#/definitions/RequestTypes"
+        },
+        "preReleaseAccessRequestSpec": {
+          "$ref": "#/definitions/PreReleaseAccessRequestSpec"
+        },
+        "creationDate": {
```

Comment:
>creationDate [](http://example.com/codeflow?start=9&length=12)

Isn't recommended property name 'createdAt'?

Reply:
Never mind it would be breaking right.

Reply:
Yes it would be breaking.